jenkins/jobs/jobs.yaml

```diff
@@ -19,7 +19,7 @@
           sudo /usr/bin/git remote update
           sudo /usr/bin/git checkout origin/master
           cd -
-          
+
```

Comment:
I don't want to remove these, but any formatting changes should really be separated out into an isolated change in future please.